core/src/services/gcs/backend.rs

```diff
@@ -119,6 +119,7 @@ pub struct GcsBuilder {
     customed_token_loader: Option<Box<dyn GoogleTokenLoad>>,
     predefined_acl: Option<String>,
     default_storage_class: Option<String>,
+    write_buffer_size: Option<usize>,
```

Comment:
Please call this buffer: `write_fixed_size` which means the fixed size that write used to flush into underlying storage.